skabase/SKALogger/SKALogger.py

```diff
@@ -235,8 +237,11 @@ def SetElementLoggingLevel(self, argin):
                 dev_proxy = DeviceProxy(element_logging_device[i])
                 dev_proxy.elementLoggingLevel = element_logging_level[i]
                 i += 1
```

Comment:
This increment needs to be removed.

Reply:
The increment is required as there could be list of devices for which logging level need to be set.
Do you suggest to implement entirely different logic to iterate on the list?

Reply:
Sorry.. Corrected. Commit #705831b58ecfa5cbf8a2fb7610047c649c603de8 addresses this.